lib/logic/permissions.dart

```diff
@@ -7,22 +7,28 @@ class Permissions {
 
   /// Whether storage permission is granted
   late PermissionStatus _permissionStorageStatus;
+  late PermissionStatus _permissionAudioStatus;
 
   /// Returns true if permissions were granted
-  bool get granted => _permissionStorageStatus == PermissionStatus.granted;
+  bool get granted => _permissionStorageStatus == PermissionStatus.granted
+      || _permissionAudioStatus == PermissionStatus.granted;
 
   /// Returns true if permissions were not granted
   bool get notGranted => !granted;
 
   Future<void> init() async {
     _permissionStorageStatus = await Permission.storage.status;
+    _permissionAudioStatus = await Permission.audio.status;
```

Comment:
maybe we should use DeviceInfo API level call storage status only `<33`, and audio status only `>=33`?

Reply:
We want to request both of them at the same time to automatically get the `audio` permission if `storage` was previously granted.

Reply:
Didn't really get what you mean, but so be it.

Reply:
Ah, sorry, I was in a hurry and didn't really read what you were writing. You are right, it is better to only check for the required permission, I changed it. What I meant is that we can (and I think we should) request both permissions simultaneously.